lgc/interface/lgc/GpurtDialect.td

```diff
@@ -304,3 +304,9 @@ def GpurtGetRayStaticIdOp : GpurtOp<"get.ray.static.id", [Memory<[(read Inaccess
   let results = (outs I32:$result);
   let summary = "get current ray static ID";
 }
+
+def GpurtGetGlobalPayloadPtrOp : GpurtOp<"get.global.payload.ptr", [Memory<[]>, WillReturn]> {
```

Comment:
It's not clear to me why we would need this. I thought we replace SPIR-V payload globals by allocas and then pass pointers to them to all the relevant lgc.rt ops (e.g., `lgc.rt.call.callable.shader` has an explicit `param` argument).

Reply:
This is needed especially for `TraceRay()` from GPURT, which we do not have the control of its parameter. But it seems that it is no longer needed after #2612 .